charts/rancher-monitoring/v0.2.3/questions.yml

```diff
@@ -1 +0,0 @@
-rancher_min_version: 2.5.9-rc1
```

Comment:
The value of `rancher_min_version` is changed to `2.6.0-rc1`, but Git treats it as a removal because the directory name is also changed. 

There is a file creation if you scroll down on this page.